tests/core_tests/test_context.py

```diff
@@ -49,7 +51,7 @@ def setUp(self):
             "project": self.project
         }
 
-        self.step = {"type":"Step", "name": "step_name", "id": 4}
+        self.step = {"type": "Step", "code": "step_name", "id": 4}
```

Comment:
Bad test data. Using wrong field to name entity. More on why this change was introduced bellow.